### ISSUE TITLE
Cannot open preferences

### ISSUE THREAD

**Issue by jrdmellow** [jorchube/monitorets#30]
I am unable to open the new Preferences window from the main app.

Running monitorets from terminal (`flatpak run io.github.jorchube.monitorets`) I can see the following error when clicking the Preferences menu item;
```
app.preferences action activated
Traceback (most recent call last):
  File "/app/share/monitorets/monitorets/main.py", line 93, in on_preferences_action
    preferences_window = PreferencesWindow(transient_for=self.props.active_window)
  File "/app/share/monitorets/monitorets/ui/preferences/preferences_window.py", line 15, in __init__
    self.add(PreferencesPageAppearance())
  File "/app/share/monitorets/monitorets/ui/preferences/preferences_page_appearance.py", line 39, in __init__
    self._layout_toggle_manager = LayoutToggleManager(self)
  File "/app/share/monitorets/monitorets/ui/layout_toggle_manager.py", line 28, in __init__
    self._set_active_toggle_for_layout(layout)
  File "/app/share/monitorets/monitorets/ui/layout_toggle_manager.py", line 31, in _set_active_toggle_for_layout
    self._layout_to_toggle_button_map[layout].set_active(True)
AttributeError: 'NoneType' object has no attribute 'set_active'
```
Monitorets version: 0.7.0
OS: Fedora Linux 37
DE: GNOME 43.2

**Comment by jorchube** [comment]
Hello!

Thank you for reporting this problem. I think I have identified the problem. Could you paste here the contents of the following file:

`~/.var/app/io.github.jorchube.monitorets/config/io.github.jorchube.monitorets/preferences.json`

If you don't know how to do that, you can execute the following command and paste here the output:

```bash
cat ~/.var/app/io.github.jorchube.monitorets/config/io.github.jorchube.monitorets/preferences.json
```

If I am under the right suspicion, as a fix if you don't want to wait until the next version I can think of two options. With monitorets **not** running:
- You can delete the `preferences.json` file I said above. After that, monitorets will start with the default settings.
- You can execute the following command:
```bash
sed -i 's/adaptive/vertical/g' ~/.var/app/io.github.jorchube.monitorets/config/io.github.jorchube.monitorets/preferences.json
```

Thank you again for reporting this! And sorry for the bug :(

**Comment by jrdmellow** [comment]
Here are the contents of the file

```
{
  "general.theme": "system",
  "general.layout": "adaptive",
  "cpu_monitor.enabled": true,
  "gpu_monitor.enabled": false,
  "memory_monitor.enabled": true,
  "downlink_monitor.enabled": true,
  "uplink_monitor.enabled": false,
  "home_usage_monitor.enabled": false,
  "root_usage_monitor.enabled": true,
  "gpu_monitor.sampling_frequency_seconds": 0.1,
  "cpu_monitor.sampling_frequency_seconds": 0.1,
  "memory_monitor.sampling_frequency_seconds": 0.1,
  "temp_monitor.acpitz-.enabled": false,
  "temp_monitor.nvme-Composite.enabled": false,
  "temp_monitor.nvme-Sensor 1.enabled": false,
  "temp_monitor.nvme-Sensor 2.enabled": false,
  "temp_monitor.coretemp-Package id 0.enabled": true,
  "temp_monitor.coretemp-Core 0.enabled": false,
  "temp_monitor.coretemp-Core 1.enabled": false,
  "temp_monitor.coretemp-Core 2.enabled": false,
  "temp_monitor.coretemp-Core 3.enabled": false,
  "temp_monitor.coretemp-Core 4.enabled": false,
  "temp_monitor.coretemp-Core 5.enabled": false,
  "temp_monitor.coretemp-Core 6.enabled": false,
  "temp_monitor.coretemp-Core 7.enabled": false
}
```
(formatted for readability)

After deleting the old `preferences.json` I can open the preferences as expected.
Thanks!

**Comment by jorchube** [comment]
Indeed thanks to you pasting the `preferences.json` content I have confirmed the cause of the issue and a fix is already in `master` (and will be soon released as version `0.7.1`).

Thanks again! :)

**Comment by jorchube** [comment]
Version `0.7.1` is out, so I consider this fixed :)
